### PR TITLE
Rebrand "AWS VSTS Tools" to "AWS Toolkit for Azure DevOps"

### PR DESCRIPTION
**Pull request by hunterwerlla**
## Related Issue(s), If Filed

#228

## Checklist

-   [ ] I have read the **README** document
-   [ ] I have read the **CONTRIBUTING** document
-   [ ] My code follows the code style of this project
-   [ ] I have added tests to cover my changes
-   [ ] A short description of the change has been added to the changelog using the script `npm run newChange`

## License

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,4 +1,4 @@
-AWS Tools for Microsoft Visual Studio Team Services
+AWS Toolkit for Azure Devops
 
 Copyright (c) 2017 Amazon, Inc
```
Comment (bryceitoc9) [inline LICENSE]: Update the date. We might want to update all of the dates with this change, in fact.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -1,30 +1,34 @@
 ---
 name: Bug report
-about: Create a bug report to help us improve the VSTS plugin
-
+about: Create a bug report to help us improve the AWS Toolkit for Azure Devops
```
Comment (bryceitoc9) [inline .github/ISSUE_TEMPLATE/bug_report.md]: It's "Azure DevOps": https://azure.microsoft.com/en-us/services/devops/

File: LICENSE (hunk 1 above)
Comment (hunterwerlla) [reply LICENSE]: I'll update all the dates with the next change I make (which touches almost all of the source files). Updated these now though.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (hunterwerlla) [reply .github/ISSUE_TEMPLATE/bug_report.md]: I'm not sure why I took a task that requires good spelling, fixed.

File: LICENSE (hunk 1 above)
Comment (hunterwerlla) [reply LICENSE]: Actually this was the only case of it (other than untouched source files which are fine to leave alone)

File: .changes/next-release/Feature-8d981d58-7fe6-4dd3-8094-25c5181eda35.json [hunk 3]
```diff
@@ -0,0 +1,4 @@
+{
+    "type": "Feature",
+    "description": "Rebraded from AWS VSTS Tools to AWS Toolkit for Azure Devops"
```
Comment (awschristou) [inline .changes/next-release/Feature-8d981d58-7fe6-4dd3-8094-25c5181eda35.json]: DevOps

**Pull request merged**